src/axom/quest/detail/MarchingCubesSingleDomain.hpp

```diff
@@ -108,6 +108,12 @@ class MarchingCubesSingleDomain
     if(m_impl) m_impl->setContourValue(m_contourVal);
   }
 
+  void setMaskValue(double maskVal)
+  {
+    m_maskVal = maskVal;
+    if(m_impl) m_impl->setMaskValue(m_maskVal);
```

Comment:
Please add curly braces around one-liners. 

Context: 
* Our current clang-format doesn't do this automatically, but it will when we upgrade beyond clang@10.  
* Our clang-tidy script does do this, but it's not run automatically.

Reply:
Done.